cls/SourceControl/Git/PullEventHandler.cls

```diff
@@ -24,14 +24,22 @@ Method OnPull() As %Status [ Abstract ]
 /// <var>pullEventClass</var>: if defined, override the configured pull event class
 ClassMethod ForModifications(ByRef files, pullEventClass As %String) As %Status
```

Comment:
Tighten with try/catch and double check exception handling of callers (make sure they're using the returned %Status)